org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/correction/MissingEnumQuickFixTest.java

```diff
@@ -109,16 +109,24 @@ public void testMissingEnumConstantDespiteDefault() throws Exception {
 		options.put(JavaCore.COMPILER_PB_MISSING_ENUM_CASE_DESPITE_DEFAULT, JavaCore.ENABLED);
 		fJProject.setOptions(options);
 		codeActions = evaluateCodeActions(cu, range);
-		assertEquals(2, codeActions.size());
+		assertEquals(3, codeActions.size());
 		Either<Command, CodeAction> codeAction = codeActions.get(0);
 		CodeAction action = codeAction.getRight();
 		assertEquals(CodeActionKind.QuickFix, action.getKind());
-		assertEquals("Add missing case statements", action.getTitle());
+		assertEquals("Add 'default' case", action.getTitle());
 		TextEdit edit = getTextEdit(codeAction);
-		assertEquals("\n        case One:", edit.getNewText());
+		// FIXME
+		assertEquals("\n        default:\n            break;", edit.getNewText());

```

Comment:
it looks like a regression no? If so is it a known bug upstream?

Reply:
I can't reproduce the issue in Eclipse. checking jdt.ls...

Reply:
Fixed.